demo/src/App.vue

```diff
@@ -48,6 +49,28 @@
       </div>
       <button id="rename-component" @click="renameComponent" :disabled="!(selectedId && renamedId)">Save</button>
     </fieldset>
+
+    <fieldset>
+      <legend>Rearrange components</legend>
+      <label>Parent component:</label>
+        <select
+            v-model="selectedAutoLayoutId"
+            name="idsAutomaticLayout"
+            id="automatic-layout-children-input"
+        >
+          <option value="">(Reorganize everything)</option>
+          <option
+              v-for="id in componentsIds"
+              :key="id"
```

Comment:
key are already use, please make it unique

Reply:
done